docs/v1.x-migration.md

```diff
@@ -120,6 +121,42 @@ m.mount(document.body, {
 });
 ```
 
+## Component arguments
+
+Passing arguments to a component hasn't changed, but how you access them within the component has changed.
```

Comment:
that's not quite right ;)
in 0.2.x you could do `m.component(myComp, 'foo')` and got `'foo'` as the second argument on the controller. This seems not working with 1.x, `attrs` always has to be an object.

But I'm not 100% sure. Please dbl-check this.